packages/context/test/acceptance/finding-bindings.ts

```diff
@@ -59,8 +59,14 @@ describe('Context bindings - Finding bindings', () => {
 
     function createTaggedBindings() {
       class Dog {}
-      ctx.bind('spot').to(new Dog()).tag('dog');
-      ctx.bind('fido').to(new Dog()).tag('dog');
+      ctx
+        .bind('spot')
+        .to(new Dog())
+        .tag('dog');
+      ctx
+        .bind('fido')
+        .to(new Dog())
+        .tag('dog');
```

Comment:
What rule control this? Isn't `ctx.bind('spot').to(new Dog()).tag('dog');` perfectly readable?

Reply:
Looks like this is hard-coded, at least I don't see any option to control this: https://github.com/prettier/prettier/blob/master/docs/options.md

Reply:
`//prettier-ignore` can be used to skip lines / blocks we want to be skipped by prettier. 

Just an option. I do think we should keep prettier formatting wherever possible to keep things consistent. 

(Also if the result of a chain is being assigned to variable then it doesn’t break it down to a command per line)